Material.Blazor/Components/AutocompleteTextField/MBAutocompleteTextField.razor

```diff
@@ -2,8 +2,8 @@
 @inherits InputComponent<string>
 
 
-<div class="@MappedClassesAndUserClasses"
-     style="@MappedStylesAndUserStyles"
+<div class="mb-autocomplete @ActiveConditionalClasses @(@class)"
```

Comment:
This is the new style of how we apply CSS classes. It's a bit more verbose, but with that also very clearly does what it says. It doesn't hide which classes are added and in which order. It should also help identifying markup issues in comparison to official docs whenever that occurs.